### PR TITLE
Add error notification toast on signup page

### LINKED ISSUE #209: Error Notification on Sign Up gets hidden on mobile UX
### Issue
When signing up on mobile device, any notification (eg password mismatch etc.) shows up at top (below heading) when you tap **Sign Up**. Since the form is long, the user is unaware of what they are missing and the app just seems stuck when you tap **Sign Up**

### Expected Outcome
The error should be shown on top of screen/bottom of screen like other notifications using standard notification system we are using (such as successful creation of project etc)

### PR DESCRIPTION
**Pull request by alicendeh**
## Summary

When signing up on mobile device, any notification (eg password mismatch etc.) shows up at top (below heading) when you tap Sign Up. Since the form is long, the user is unaware of what they are missing and the app just seems stuck when you tap Sign Up

## Fix
The error is  shown on top of screen  like other notifications using standard notification system

Closes #209

## Changes

 - Added a toast in Signupscript

## Screenshots 

Screenshot from 2022-04-18 17-09-32

### CODE REVIEW

**Review by tuxology — APPROVED** [review]
LGTM :rainbow_flag: Will wait for @NdibeRaymond to merge this. I haven't tested but it looks simple.

**Comment by alicendeh** [comment]
OKay sure @tuxology . Waiting for @NdibeRaymond then

File: zubhub_frontend/zubhub/src/views/signup/signupScripts.js [hunk 1]
```diff
@@ -91,6 +92,10 @@ export const signup = (e, props) => {
             });
           }
         });
+    } else {
+      toast.error(
```
Comment (NdibeRaymond) [inline zubhub_frontend/zubhub/src/views/signup/signupScripts.js]: hello @alicendeh thanks for submitting this PR!
Few changes and we can merge this. It looks like you aren't localizing messages here. Here is what you can do about it:
* Search for "toast" on your code editor and observe how it's being used in other places.
* Create entries for this error message in the signup section of each locale file (en/translation.json and hi/translation.json) then add the message to the en file.
* search for english to hindi on google and use the provided translator to covert the english translation to  hindi then paste in in the corresponding location in the hi/translation.json file

**Review by NdibeRaymond — CHANGES_REQUESTED** [review]

File: zubhub_frontend/zubhub/src/views/signup/signupScripts.js (hunk 1 above)
Comment (alicendeh) [reply zubhub_frontend/zubhub/src/views/signup/signupScripts.js]: Oh yes, I completely forgot the system is a bilingual system. My bad.
Let me modify that .

**Comment by alicendeh** [comment]
Done @NdibeRaymond. Please have a look

**Review by NdibeRaymond — APPROVED** [review]

**Pull request merged**